src/Microsoft.ML.Data/Utilities/ModelFileUtils.cs

```diff
@@ -25,6 +25,8 @@ internal static class ModelFileUtils
     {
         public const string DirPredictor = "Predictor";
         public const string DirDataLoaderModel = "DataLoaderModel";
+        public const string DirTransformerChain = "TransformerChain";
```

Comment:
(minor suggestion)

Instead of hard-coding these strings here, can you use/reference the existing const strings for these?

```
private const string DirTransformerChain = TransformerChain.LoaderSignature;
private const string SchemaEntryName =ModelOperationsCatalog.SchemaEntryName; 
```
Note that you will need to make ModelOperationsCatalog.SchemaEntryName internal, since it is private right now.